src/test/scala/nl.knaw.dans.easy.bagstore/server/StoresServletSpec.scala

```diff
@@ -328,8 +333,15 @@ class StoresServletSpec extends TestSupportFixture
     }
   }
 
+  def authenticationHeader(username: String, password: String, authType: String = "Basic"): List[(String, String)] = {
+    val encoded = Base64.getEncoder.encodeToString(s"$username:$password")
+    List("Authorization" -> s"$authType $encoded")
+  }
```

Comment:
Ah, I suppose that is how it should be done in the deposit-api [TestSupportFixture](https://github.com/DANS-KNAW/easy-deposit-api/blob/3768cf999950ac6b5bedb8f332c7d51b3c0bb436/src/test/scala/nl.knaw.dans.easy.deposit/TestSupportFixture.scala#L45)

Reply:
👍 Make it so :)